input/escape.cfg

```diff
@@ -80,8 +80,8 @@ F_crit                      = 0.2       # Critical flux, below which the model w
 # Atmospheric escape 
 escape_model                = 1         # Escape model to be used, 0: None | 1: ZEPHYRUS | 2: Dummy
 escape_stop                 = 3e-4      # Terminate when atm mass drops below this fraction of its initial mass
-escape_dummy_rate           = 0.0       # Bulk escape rate for dummy escape model [kg s-1]
-escape_el_rate              = 0.15      # Efficiency factor for energy-limited escape (varies typically 0.1 < epsilon < 0.6) [dimensionless]    
+escape_dummy_rate           = 4e3       # Bulk escape rate for dummy escape model [kg s-1]
```

Comment:
I don't think it is relevant to define an escape dummy rate here since your are not using the dummy model but Zephyrus instead.

Reply:
Yes I removed the line for escape_dummy_rate.

Reply:
Also, in a previous commit on the escape2 branch, I changed the name of the escape_el_rate variable to efficiency_factor. I also changed it into the start_proteus.py file (line 479).